angular/app/pages/landing/landing.scss

```diff
@@ -98,7 +96,7 @@ body{
 .DemoCode{
     font-family: "Lucida Console", Monaco, monospace;
     overflow: auto !important;
-    padding: 10px;
+    padding: 10px 10px 10px 20px;
```

Comment:
The CSS lint plugin for PHPStorm was complaining about this. I think because we had padding:10px; defining all sides, and then the left only being overwritten it was considered to be a duplicate style. I was just trying to get ride of the yellow... lol.

I think this is the best approach though in the end. We can do everything in one line rather than spread padding rules across two lines. It's not a huge deal - I did all of these little CSS fixes in a separate commit for this reason. I'm going to take most of this boilerplate stuff out when I complete my site today.